aa_py_openpyxl_util/_open_locked_file.py

```diff
@@ -0,0 +1,166 @@
+import os
+from typing import Callable, Union, Tuple, IO, Any, Optional
+import os
+from pathlib import Path
+
+
+open_locked = open
+
+if os.name == "nt":
```

Comment:
@heetbeet this makes our package untestable on Linux, which breaks CI. We need to implement a dummy when not running on Windows. Preferrably check this within the class rather than wrap the class in `if`.